x/staking/keeper/keeper.go

```diff
@@ -22,13 +22,14 @@ var _ types.DelegationSet = Keeper{}
 
 // keeper of the staking store
 type Keeper struct {
-	storeKey           sdk.StoreKey
-	cdc                *codec.Codec
-	supplyKeeper       types.SupplyKeeper
-	hooks              types.StakingHooks
-	paramstore         params.Subspace
-	validatorCache     map[string]cachedValidator
-	validatorCacheList *list.List
+	storeKey              sdk.StoreKey
+	cdc                   *codec.Codec
+	supplyKeeper          types.SupplyKeeper
+	hooks                 types.StakingHooks
+	paramstore            params.Subspace
+	validatorCache        map[string]cachedValidator
+	validatorCacheList    *list.List
+	delayValidatorUpdates bool
```

Comment:
Allow delaying of validator updates to be turned off for tests.

Reply:
I guess we might need to go through and update the tests in the fullness of time